imagine/main.go

```diff
@@ -134,6 +137,10 @@ func (c *Config) readSpecs() error {
 }
 
 func main() {
+	go func() {
+		fmt.Printf("failed to start pprof server on 6060: %v\n", http.ListenAndServe("localhost:6060", nil))
```

Comment:
while i agree that this is probably correct, i might find it easier to read with the `if err := http.ListenAndServe(...); err != nil {` around the printf, just because otherwise it takes a while to realize why this doesn't unconditionally print the message.